src/LocalDateRange.php

```diff
@@ -244,6 +244,25 @@ public function jsonSerialize(): string
         return (string) $this;
     }
 
+    /**
+     * Converts this LocalDateRange to Interval instance.
+     *
+     * The result is Interval from 00:00 start date and 00:00 end date + one day (because end in Interval is exclude)
+     * in the given time-zone.
+     */
```

Comment:
```suggestion
    /**
     * Converts this LocalDateRange to an Interval in the given TimeZone.
     *
     * The result is an Interval from 00:00 on the start date to 00:00 on the end date + one day
     * (the end Instant of Interval being exclusive).
     */
```

Reply:
Also, should we add a warning relative to [this](https://github.com/brick/date-time/blob/0.5.3/src/ZonedDateTime.php#L60-L86)?

As to what happens if `00:00` falls within a DST transition (I'm not even sure if this is possible in any timezone).